### PR TITLE
fix: history location purge

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Fix history location purge API async mechanism
- [x] Introduce scroll to anchor for blockchain balance

### CODE REVIEW

**Comment by yabirgb** [comment]
Loopring scroll works flawless :+1:

File: frontend/app/src/store/history/actions.ts [hunk 1]
```diff
@@ -219,7 +219,25 @@ export const actions: ActionTree<HistoryState, RotkehlchenState> = {
       onlyCache: false
     };
 
-    await api.history.trades(params);
+    const { awaitTask } = useTasks();
```
Comment (kelsos) [inline frontend/app/src/store/history/actions.ts:222]: We not trigger the action for fetching with only cache false? Instead of copying

**Review by yabirgb — APPROVED** [review]
Tested both the location purge and the loopring scroll and seems to work for me. I give my approval pending to see any comments on the code from kelsos

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**